spec/requests/admin_payroll_runs_spec.rb

```diff
@@ -32,23 +32,6 @@
 
         expect(response).to redirect_to(admin_payroll_run_path(payroll_run))
       end
-
-      context "when a payroll run contains claims from the same person with different bank details" do
-        it "doesn’t create a payroll run and shows an error" do
```

Comment:
I feel a little uneasy about losing this spec. We rely on other parts of the system preventing us from getting in this state, but those parts could change. However it's not likely to change so if leaving this spec in causes complication I'm happy for it to be removed.